rules/typescript.js

```diff
@@ -18,7 +18,11 @@ module.exports = {
     '@typescript-eslint/no-unused-vars': 'error',
     '@typescript-eslint/no-useless-constructor': 'error',
     '@typescript-eslint/prefer-for-of': 'error',
-    '@typescript-eslint/prefer-optional-chain': 'error',
+    '@typescript-eslint/prefer-optional-chain': 'off',
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/camelcase": "off",
+    "@typescript-eslint/no-var-requires": "off",
```

Comment:
I don't think this should be off, this should be a warning instead

Reply:
Hmm, it's discussable. On the one hand yes, probably it should throw a warning, instead of being off. But on the other hand we never do anything with warnings, so they are kinda useless

Reply:
Well, I do try to change code I write that generates warnings, only if it is not possible or very difficult like with some cyclic imports, then I ignore it. But, I don't have a strong opinion about those 2 cases

Reply:
Enabling warnings for the first two rules gives 175 warnings for the LP 😱

Reply:
I guess leave them off for now then. Most of these will go away after strict mode is on

Reply:
I added a "Eslint warnings vs errors" topic for the next weekly. I personally believe there should be no eslint warnings. But we can discuss it together to hear everyone's opinion.

Reply:
Yeah, I don't really have a strong opinion about this. I am fine either way

Reply:
I would leave these rules off.